app/models/concerns/cms/addon/sns_poster.rb

```diff
@@ -135,17 +136,16 @@ def post_to_facebook(snskeys)
         if file_ids.present?
           image_path = files.first.full_url
         end
-        graph = access_token_facebook(snskeys)
+#       graph = access_token_facebook(snskeys)
+        graph = Koala::Facebook::API.new(@cur_site.facebook_access_token)
         # facebokに投稿し、戻り値を取得
         facebook_params = graph.put_wall_post(
-          message,
-          {
-            "name"=> "#{name} - #{site.name}",
-            "link"=> full_url,
-            "picture"=> image_path,
-            "description"=> description
-          }
-        )
+          message, {
+          "name"=> "#{name} - #{site.name}",
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

<!-- Comment by [SideCI](https://sideci.com) -->